### PR TITLE
Updated multiple scheduler beta docs.

### LINKED ISSUE #2304: Update document  for multiple-scheduler
**This is a...** 
Bug Report

**Problem:**
Because of [kubernetes#38871](https://github.com/kubernetes/kubernetes/pull/38871) , we need to update multiple scheduler document to use `PodSpec.SchedulerName` instead of annotation,

**Page to Update:**
https://kubernetes.io/docs/admin/multiple-schedulers/

/cc @davidopp @timothysc

### PR DESCRIPTION
**Pull request by k82cn**
fixes #2304

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2378)

### CODE REVIEW

File: docs/admin/multiple-schedulers.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ config. Save it as `my-scheduler.yaml`:
 {% include code.html language="yaml" file="multiple-schedulers/my-scheduler.yaml" ghlink="/docs/admin/multiple-schedulers/my-scheduler.yaml" %}
 
 An important thing to note here is that the name of the scheduler specified as an
-argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `scheduler.alpha.kubernetes.io/name` annotation on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the `spec.schedulername` on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
```
Comment (k82cn) [inline docs/admin/multiple-schedulers.md]: Done :).

**Comment by devin-donnelly** [comment]
It looks like this was merged early for 1.6.

**Comment by k82cn** [comment]
#2588 is second patch to update field name in 1.6; already in 1.6 milestone.

**Pull request merged**